linux - sessions - module - closes #603 (#609)
bob-crutchley/courseware: topics/linux/modules/sessions/README.md

### Common Commands
Below are some basic fact finding commands which will be useful when using the command line for the first time.

```bash
ls [options]        #lists things in current directory
ps [options]        #shows currently active processes
echo [arguments]    #displays arguments on the screen
who                 #lists users currently logged in to the system
uname [options]     #displays basic OS information
pwd                 #displays current working directory on the screen
```

### Working with text files
There are also commands which are useful when working with text files
```bash
cat [object]    #displays content in a file, does not pause
grep [argument] #displays lines which contain a specified pattern
wc -l [object]  #counts the lines in a file
tail [object]   # shows the last few lines in a file
head [object]   #shows the first few lines in a file
more [object]   #like cat, but pauses after each screen
```

## Getting help in Linux

### CLI help
Linux has a few features that are very helpful if you don't quite understand what a command does.

> The `man` command will bring up the local documentation for the command that you query. e.g.

Reviewer: ```suggestion
The `man` command opens up the manual which is the documentation about the command you are inquiring. e.g.
```

```bash
man grep
```

> The `--help` option will display the information from the documentation on the screen

Reviewer: ```suggestion
This example would open up the manual for the `grep` command. You would be able to see the options available for the command as well as description.

The `--help` option will display the information from the documentation on the screen
```
